app/templates/components/job-infrastructure-notification.hbs

```diff
@@ -7,17 +7,23 @@
     {{/if}}
     {{#if isPreciseEOL}}
       {{#notice-banner}}
-        This job {{conjugatedRun}} on our <b>Precise</b> environment, which is in the process of being decommissioned.  Please update to a newer Ubuntu version by specifying <code>dist: xenial</code> in your <em>.travis.yml</em>.
+        This job {{conjugatedRun}} on our <b>Precise</b> environment, 
+        which is in the process of being decommissioned.  
+        Please update to a newer Ubuntu version by specifying <code>dist: xenial</code> in your <em>.travis.yml</em>.
```

Comment:
The changes above are just formatting (split into multiple shorter lines)

Reply:
Thanks for clarifying - I think that makes them easier to parse. yes!